ic-os/guestos/context/docker-base.prod

```diff
@@ -1 +1 @@
-ghcr.io/dfinity/guestos-base@sha256:0a485d2d033df5aa7efd4ccbaad432c29d18f0f165cdcbeed5d49790c0d16340
+ghcr.io/dfinity/guestos-base@sha256:a30e912b3df373ebaa1ce0914b8769295f5365e9d8acb13dc08b4932ba2770c5
```

Comment:
This contains changes for the GuestOS 24.04 upgrade, we'll keep this in an RC for the initial rollout